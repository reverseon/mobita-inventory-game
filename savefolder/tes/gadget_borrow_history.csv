id;id_peminjam;id_gadget;tanggal_peminjaman;jumlah;is_returned
1;38070;G3;23/10/2021;1;true
2;38070;G1;23/09/2020;200;false
3;38070;G4;23/10/2019;200;false
4;38070;G3;23/10/2018;1;true
5;38070;G1;23/09/2017;200;false
6;38070;G4;23/10/2016;200;true
7;38070;G3;23/10/2015;1;true
8;38070;G1;23/09/2014;200;false
9;38070;G4;23/10/2013;200;true
10;38070;G3;23/10/2012;1;false
11;38070;G1;23/09/2011;200;true
12;38070;G4;23/10/2010;200;true
