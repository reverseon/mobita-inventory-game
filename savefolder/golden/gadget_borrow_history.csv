id;id_peminjam;id_gadget;tanggal_peminjaman;jumlah;is_returned
76853;38070;G11;12/12/2001;0;true
108662;53615;G5;20/10/2004;10;false
